index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  <link rel="stylesseet" href="styles.css" >
+  </head>
+
+  <body>
+<section id ="welcome-section">
+  <h1> Texto </h1>
+</section>
+<section id ="projects">
+  <h2 class="project-tile"> Projects</h2>
+  <a href="www.google.com"> link</a>
+  </section>
+  <nav id="navbar">
+    <a href="#welcome-section"> link dentro de la pagina</a>

```

Comment:
🙌 Please remember to change the language inside the label to English 😜

```suggestion
    <a href="#welcome-section">Link inside the web page.</a>

```